other_recipes/fried rice.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+   <h1>Fried Rice</h1>
+   <img src="American_fried_rice.jpg"width="30%">
+   <h1>Description</h1>
+   <p> Fried rice is a dish of cooked rice that has been stir-fried in a wok or a frying pan and is usually mixed with other ingredients</p>
+   <h1>Ingredients</h1>
```

Comment:
- [ ] Please replace the other `h1` element with any other `h2, h3, h4, or h5` element because it helps improve your page SEO score. Read about [SEO optimization here](https://www.wordstream.com/seo).